### PR TITLE
init_model/27494 relationship

### PR DESCRIPTION
**Pull request by tranduythuat**
**Redmine URL**
- https://edu-redmine.sun-asterisk.vn/issues/27494

**Work**
- init model 
- definite relationship

### CODE REVIEW

File: app/Models/Social.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class Social extends Model
+{
+    protected $fillable = [
+        'provider_id',
+        'provider_name',
+        'access_token',
+        'created_at',
+        'update_at'
+    ];
```
Comment (phuongsai) [inline app/Models/Social.php:15]:
```suggestion
    'user_id',
    ];
```

File: app/Models/Question.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+class Question extends Model
+{
+    protected $fillable = [
+        'name',
+        'lesson_id',
+        'created_at',
+        'update_at',
+        'created_at',
+        'update_at'
```
Comment (phuongsai) [inline app/Models/Question.php:17]:
```suggestion

```

**Comment by tranduythuat** [comment]
ready

**Review by phuongsai — CHANGES_REQUESTED** [review]
Code chưa OK cần fix lại cho hợp lí.

**Comment by tranduythuat** [comment]
ready

fixed

File: app/Models/User.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'email_verified_at' => 'datetime',
+    ];
+
+    public function socials()
```
[the remark this answers is not in the document]
Comment (phuongsai) [reply app/Models/User.php:50]: Chưa fix theo "Naming Conventions" #3

**Pull request merged**